config/webpack.config.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const MinifyPlugin = require('babel-minify-webpack-plugin');
+
+const staticConfig = require('./webpack.static.config');
+const themeableConfig = require('./webpack.themeable.config');
+
+const commonConfig = {
+  // 'source-map' raises errors (something about the combination)
+  // of MinifyPlugin and ExtractTextPlugin) -- see:
+  // https://github.com/webpack-contrib/babel-minify-webpack-plugin/issues/68
+  devtool: 'cheap-source-map',
```

Comment:
I originally picked babel-minify (nee babili) over uglify because uglify (at the time -- last summer) came with some nasty compatibility issues. I'll give uglify a shot and see if it does any better.